code/modules/unit_tests/metabolizing.dm

```diff
@@ -51,15 +51,15 @@
 	var/mob/living/carbon/human/syringe_user = allocate(/mob/living/carbon/human/consistent)
 	var/mob/living/carbon/human/pill_syringe_user = allocate(/mob/living/carbon/human/consistent)
 
-	var/datum/mind/pill_mind = new /datum/mind("Mothcocks")
+	var/datum/mind/pill_mind = new /datum/mind(null)
```

Comment:
RIP, this was Floyd's 😢 

Also any reason this can't turn into `new` without any other fluff?

Reply:
No. I tend to err on the side of implicit when I have intent (I.e. I intend to create a ckeyless mind) but no reason it can't just be empty.